nifi-nar-bundles/nifi-site-to-site-reporting-bundle/nifi-site-to-site-reporting-task/src/main/java/org/apache/nifi/reporting/AbstractSiteToSiteReportingTask.java

```diff
@@ -190,7 +191,7 @@ public void reportEvent(final Severity severity, final String category, final St
                 context.getProperty(HTTP_PROXY_USERNAME).getValue(), context.getProperty(HTTP_PROXY_PASSWORD).getValue());
 
         siteToSiteClient = new SiteToSiteClient.Builder()
-                .url(destinationUrl)
+                .urls(SiteToSiteRestApiClient.parseClusterUrls(destinationUrl))
```

Comment:
This change just reminded me that `SiteToSiteRestApiClient.parseClusterUrls` also validates the destinationUrl. Probably it'd be a better approach to use `SiteToSiteRestApiClient.parseClusterUrls` and catch IllegalArgumentException from `NiFiUrlValidator.validate()`, instead of implementing different validation code there. How do you think?

Reply:
Realizing that this method is adding /nifi-api (or converting /nifi into /nifi-api). I changed the validator and updated the unit test accordingly.